### ISSUE TITLE
Verify Compare-Module version comparison

### ISSUE THREAD

**Issue by jdhitsolutions** [jdhitsolutions/PSScriptTools#21]
This was posted on the gist for the original function. Need to verify I have addressed this:

Thanks a lot for your Function,! but you compare version strings and not Version Objects.
Therefore, your code does not recognize that Version 11 is newer than Version 2:

C:\Temp> '2.0.0.0' -gt '10.0.0.0'
true # For versions, this result is wrong
This works:

C:\Temp> [version]'2.2.3.4' -gt [version]'11.2.3.4'
False

**Comment by jdhitsolutions** [comment]
As far as I can tell my code is comparing the version property as a version object and not as strings. But I made a few performance tweaks regardless in v1.8.1

**Comment by jdhitsolutions** [comment]
comp

**Comment by jdhitsolutions** [comment]
Wrong. Modified comparison to explicitly cast as version.

```powershell
foreach ($item in $installedVersions) {
                        If ([version]$_.Version -gt [version]$item) {
                            $result = $True
                        }
                        else {
                            $result = $False
                        }
                    }
```

**Comment by jdhitsolutions** [comment]
fix released in v1.8.1
